wsdl/ver10/schema/onvif.xsd

```diff
@@ -1552,6 +1561,12 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 				</xs:annotation>
 			</xs:element>
 			<xs:element name="Extension" type="tt:PTZStatusFilterOptionsExtension" minOccurs="0"/>
+			<xs:element name="FieldOfView" type="xs:boolean" minOccurs="0">
```

Comment:
Why this FieldOfView is defined in both MetadataConfigurationOptions and PTZStatusFilterOptions

Reply:
Good question. 
This is the first time I have tried to work with the onvif specs so I do appreciate some guidance. 

I tried to fix it now, but not certain if it became completely correct?